zerver/openapi/zulip.yaml

```diff
@@ -3240,6 +3240,14 @@ paths:
                                     [register](/api/register-queue) responses.
 
                                     **Changes**: New in Zulip 4.0 (feature level 59).
+                                zulip_merge_base:
+                                  type: string
+                                  description: |
+                                    The Zulip merge base number, in the format where this appears
+                                    in the [server_settings](/api/get-server-settings) and
+                                    [register](/api/register-queue) responses.
+
+                                    **Changes**: New in Zulip 5.0 (feature level 87).
```

Comment:
Should this be 88?

Reply:
Oh yes, missed by Tim while resolving conflicts.